### PR TITLE
64 bit support for simulators

### PR DESCRIPTION
**Pull request by esavage13**

### CODE REVIEW

**Comment by esavage13** [comment]
With respect to the server changes:
* 8a6a5f8 - changed the server from mobileproxy to tags (based on our understanding at that time)
* 75ebe9e - reverted those server changes after a conversation with your team

With respect to the enabling of 64 bit for the simulator:
* 8a6a5f8 contains the following lines to enable that:
```
ARCHS = "$(ARCHS_STANDARD)";
...
VALID_ARCHS = "arm64 armv7 armv7s i386 x86_64";
```

**Comment by burkaygur** [comment]
Looks good! @Nymbil @mouli-bluekai

**Comment by shunchu** [comment]
Playing devil's advocate here: Can someone update the `CHANGELOG.md` file as well so that another developer trying to upgrade to this version can, at a glance, find out what changed and if it's worth the effort for him/her to upgrade?

File: sdk/Bluekai_SDK.xcodeproj/project.xcworkspace/xcshareddata/Bluekai_SDK.xccheckout [hunk 1]
```diff
@@ -10,31 +10,31 @@
 	<string>Bluekai_SDK</string>
 	<key>IDESourceControlProjectOriginsDictionary</key>
 	<dict>
-		<key>6AAA46BB-AB0D-4DE0-9529-E655CEDE5808</key>
-		<string>ssh://github.com/BlueKai/bluekai-ios-sdk.git</string>
+		<key>CA4AC5F629E6FDEDF1C60AD4967190C8B0F4B001</key>
+		<string>github.com:esavage13/bluekai-ios-sdk.git</string>
```
Comment (scottt2) [inline sdk/Bluekai_SDK.xcodeproj/project.xcworkspace/xcshareddata/Bluekai_SDK.xccheckout]: Is this correct? Do we want that merged into master on our repo?

**Comment by esavage13** [comment]
Just reverted Bluekai_SDK.xccheckout to its state as of 539ac35.
I doubt we need this as git maintains its own state and Xcode changes this as needed anyway.

**Comment by scottt2** [comment]
Cool.

-Tyler

**Comment by codyrotwein** [comment]
This is still an issue in the latest sdk. Can't build on simulator (iPhone 6s 9.2).

**Comment by burkaygur** [comment]
@codyrotwein Check the latest prerelease version 2.0.4

https://github.com/BlueKai/bluekai-ios-sdk/blob/master/lib/bluekai-ios-static-ARC-v2.0.4.zip

**Comment by codyrotwein** [comment]
That fixed the error, thanks.

**Comment by antonbk** [comment]
After v3 updates this is pull request can no longer be merged.

**Pull request closed without merging**